docs/faq/glossary.md

```diff
@@ -0,0 +1,63 @@
+
+## FDD - Fraud Detection & Defense
+FDD is part of GitcoinDao and is responsible for detecting and preventing Sybil attacks against Gitcoin Grants.
+FDD uses lego to fight Sybil's attacks some of them include Gitcoin Passport, SDD, and human review.
+
+You can find more resources on [Gitcoin blog](https://gov.gitcoin.co/t/introducing-the-fdd-review/11095).
+
+## Gitcoin Passport
+Gitcoin Passport is one of the Sybil defense lego built by Gitcoin. It allows a user to acquire stamps to prove his humanness. These stamps are proof of either on-chain activities such as a lens profile, or a POAP and off-chain hard-to-fake web2 actions such as a Twitter account with several followers, a GitHub account with many stars or contributions, a proof of identity with for example BrightId. These credentials are stored on the Ceramic and can be queried to build a trust score for an address.
+
+Verify your credentials with Passport [here](https://passport.gitcoin.co/).
+
+
+## Stamps
+Stamps are credentials or proof of activities that can be acquired by a user by connecting and verifying ownerships of accounts both with Gitcoin Passport.
+
+## SAD - Sybil Account Detection
+SAD is a Sybil defense lego developed by Gitcoin that looks at the on-chain history of a grant contributor to score the EOA of the user. 
+
+## Gitcoin
+ Gitcoin describes itself as a platform “to fund builders looking for meaningful, open source work”. Their mission is to build a more open, collaborative, and economically empowering world. By giving tools to communities, they help build and funds public goods and bring the future digital public infrastructure. 
+
+## Squelched
+We say that a user is "squelched" when he does not have his funds matched by quadratic funding because he has been detected as a Sybil attacker. 
+
+## POAP  - Proof of Attendance Protocol
+POAP is commonly used to describe a collectible that can be used, for example, to certify the presence in a physical or online event, the participation in a hackathon. Broadly, it describes the gift of an event organizer to people attending the event.
+You can find more details about POAP in the [documentation](https://poap.zendesk.com/hc/en-us/articles/9494654007437-What-Is-POAP-)
+
+## Lego
+A Lego is a composable unit that performs a definite action. You can see it as a piece of software in which, for a given input, it will return an output. Gitcoin Passport scorer is an example of Sybil detection lego developed by Gitcoin that, from the stamps verified by a user returns a trust score.
+
+You can find more details in this Gitcoin blog post: [Anti-Sybil Legos](https://gov.gitcoin.co/t/anti-sybil-legos/12265)
+
+## What is Exploratory Data Analysis (EDA)
+(**V0.0 @G-r-ay : Review needed**)
```

Comment:
Should be removed

Reply:
Is it redundant, wrong, or something else?

Reply:
Take a look at #38 as these two PRs can be merged together now.

Reply:
I think clarification is that the V0.0 is no longer needed in this version. Let me make an update and push to update this.

Also @sarob reviewed #38 and looking good. Thank you for the update!